src/patternfly/components/Select/select.scss

```diff
@@ -224,6 +234,7 @@
   // Max-width is needed for ellipsing the text. This leaves enough space for the toggle caret.
   // However, for cases where the clear icon button or a badge are added, the correct width will need to be calculated in script
   max-width: var(--pf-c-select__toggle-wrapper--MaxWidth);
+  padding-top: var(--pf-c-select__toggle-wrapper--PaddingTop);
```

Comment:
If I understand this part correctly, I think you can remove this and just add `padding-top` in the `m-typeahead` block. We usually don't add a property/variable that doesn't do anything, just so it can be overwritten later in a modifier.